addons/mrp/models/mrp_bom.py

```diff
@@ -46,8 +46,7 @@ def _get_default_product_uom_id(self):
     product_uom_id = fields.Many2one(
         'uom.uom', 'Unit of Measure',
         default=_get_default_product_uom_id, required=True,
-        help="Unit of Measure (Unit of Measure) is the unit of measurement for the inventory control", domain="[('category_id', '=', product_uom_category_id)]")
-    product_uom_category_id = fields.Many2one(related='product_tmpl_id.uom_id.category_id')
+        help="Unit of Measure (Unit of Measure) is the unit of measurement for the inventory control")
```

Comment:
I guess that `help` could be re-written. :laughing: 
But that can be done at a later point